$NetBSD: patch-libstdc++-v3_configure.ac,v 1.1 2022/10/17 13:25:31 js Exp $

Don't check for dlopen in libstdc++-v3 - it causes a link test after
GCC_NO_EXECUTABLES.

--- libstdc++-v3/configure.ac.orig	2021-07-28 06:55:09.228314429 +0000
+++ libstdc++-v3/configure.ac
@@ -90,9 +90,6 @@ AC_SYS_LARGEFILE
 GLIBCXX_CONFIGURE
 
 # Libtool setup.
-if test "x${with_newlib}" != "xyes" && test "x${with_avrlibc}" != "xyes"; then
-  AC_LIBTOOL_DLOPEN
-fi
 AM_PROG_LIBTOOL
 ACX_LT_HOST_FLAGS
 AC_SUBST(enable_shared)
